tools/seed/seed-users-and-fleets.sh

```
@@ -0,0 +1,326 @@
+}
+
+# Helper: create user via /users/admin (ignores errors if user already exists)
+# Sets LAST_USER_CREATED=1 if created, 0 if skipped.
+LAST_USER_CREATED=0
+create_user() {
+  local data="$1"
+  local name email
+  name=$(echo "$data" | python3 -c "import sys,json; d=json.load(sys.stdin); print(d['name'])")
+  email=$(echo "$data" | python3 -c "import sys,json; d=json.load(sys.stdin); print(d.get('email',''))")
+  local result
+  result=$(curl -X POST $CURL_FLAGS -H "$AUTH" -H "Content-Type: application/json" "$API/users/admin" --insecure -d "$data" 2>&1)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)